src/main/java/org/codehaus/mojo/versions/AbstractVersionsUpdaterMojo.java

```diff
@@ -417,10 +421,61 @@ protected abstract void update( ModifiedPomXMLEventReader pom )
      * @return <code>true</code> if the update should be applied.
      * @since 1.0-alpha-1
      */
+    @Deprecated
     protected boolean shouldApplyUpdate( Artifact artifact, String currentVersion, ArtifactVersion updateVersion )
```

Comment:
the deprecated method should call the new with correct parameter. We should avoid too much duplicated code.

Reply:
Good point Olamy. Appreciate your patience with me.

Reply:
Pushed the changes. Thanks